src/Spryker/Client/RabbitMq/RabbitMqConfig.php

```diff
@@ -75,34 +106,89 @@ protected function getQueueConnectionConfigs(): array
      */
     protected function getQueueOptions()
     {
-        $queueOptionCollection = new ArrayObject();
-        $queueOptionCollection->append(new RabbitMqOptionTransfer());
+        if ($this->queueOptionCollection !== null) {
+            return $this->queueOptionCollection;
+        }
 
-        return $queueOptionCollection;
+        $queueConfigurations = $this->getQueueConfiguration();
+        $this->queueOptionCollection = new ArrayObject();
+
+        foreach ($queueConfigurations as $queueNameKey => $queueConfiguration) {
+            if (!is_array($queueConfiguration)) {
+                $this->queueOptionCollection->append($this->createQueueOption($queueConfiguration, sprintf('%s.error', $queueConfiguration), 'error'));
```

Comment:
It's a bit scary that the core module create OOTB `error` queues, I expect the `error` or any other namings come form project not core

Reply:
In this case the best option to store RabbitMqConfig::getQueueOptions() on the project level